binding.gyp

```diff
@@ -59,6 +59,34 @@
 					},
 					"libraries": [ '<(SOLETTA_LIBS)' ],
 					"dependencies": [ "collectbindings" ]
+				},
+				{
+					"target_name" : "generatecythonbindings",
```

Comment:
do you really want to use node.js build system? Any benefits or just following what they did? For us it would be nicer to integrate in the host buildsystem (kconfig/makefiles). Gabriel used gyp to benefit from npm/node support.